.circleci/config.yml

```diff
@@ -167,29 +167,17 @@ jobs:
           at: ~/app
       - restore_cache:
           key: dependency-cache-{{ checksum "package-lock.json" }}
-      - run:
-          name: Run the node app.
-          command: npm run start:e2e
-          background: true
-      - run:
-          name: Get wiremock
-          command: curl -o wiremock.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-standalone/2.27.2/wiremock-standalone-2.27.2.jar
-      - run:
-          name: Run wiremock
-          command: java -jar wiremock.jar --port 9091
-          background: true
```

Comment:
now running wiremock through docker compose and don't need to worry about running the app

Reply:
currently rebuilds docker image rather than using the newly built one - would be nice to fix this